config.m4

```diff
@@ -170,6 +170,12 @@ if test "$PHP_MONGODB" != "no"; then
     src/MongoDB/Monitoring/functions.c \
   "
 
+  PHP_ARG_WITH([system-libs],
```

Comment:
I think we should prefix this with `mongodb-` so we don't pollute the global namespace if our build process was invoked from a bundled PHP build. This was realistically never an issue for the old "libmongoc" and "libbson" names, but "system-libs" is much more generic.

Reply:
Good point 👍